### ISSUE TITLE
Fix Atom feed

### ISSUE THREAD

**Issue by Minoru** [adius/adius.github.io#12]
Hi! At the moment, the Atom feed at https://adriansieber.com/atom.xml looks like this:

```xml
<?xml version="1.0" encoding="utf-8"?>
<feed xmlns="http://www.w3.org/2005/Atom">
	<title type="text" xml:lang="en">Adrian Sieber</title>
	<link
		type="application/atom+xml"
		href="http://adriansieber.com/atom.xml"
		rel="self"
	/>
	<link
		type="text/html"
		href="http://adriansieber.com/"
		rel="alternate"
	/>

	<updated>{{ site.time | date_to_xmlschema }}</updated>
	<id>adriansieber.com</id>
	<author>
		<name>Adrian Sieber</name>
	</author>
	<rights>Copyright by Adrian Sieber. All rights reserved.</rights>

	{% for post in site.posts limit:20 %}
	<entry>
		<title>{{ post.title }}</title>
		<link href="http://adriansieber.com{{ post.url }}"/>
		<updated>{{ post.date | date_to_xmlschema }}</updated>
		<id>http://adriansieber.com{{ post.id }}</id>
		<content type="html">{{ post.content | xml_escape }}</content>
	</entry>
	{% endfor %}
</feed>
```

Looks like the template wasn't processed and is served as a file.

**Comment by tfausak** [comment]
👍 I also ran into this problem.

**Comment by ad-si** [comment]
Oh damn, I missed this issue. Thanks for noticing!!! 
Should have been fixed in ea9d0df8738a1c644befb314da6fafbceb45bcfe
